src/app/repository/project-repository.service.ts

```diff
@@ -0,0 +1,40 @@
+import {Injectable} from '@angular/core';
+import {CpeApiService, iCpeListResource} from "./cpe-api.service";
+import {Project, CpeResource} from "./repository-model";
+import {firstValueFrom, lastValueFrom} from "rxjs";
+import {HttpClient} from "@angular/common/http";
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProjectRepositoryService {
+
+  constructor(private apiService: CpeApiService, private http: HttpClient) {
```

Comment:
HttpClient is not used in this component and can be removed.

Reply:
removed